nix/nomadChart.nix

```diff
@@ -0,0 +1,240 @@
+                      "traefik.http.routers.cardano-graphql-${namespace}.rule=Host(`graphql.${domain}`) && PathPrefix(`/`)"
+                      "traefik.http.routers.cardano-graphql-${namespace}.entrypoints=https"
+                      "traefik.http.routers.cardano-graphql-${namespace}.tls.certresolver=acme"
+                    ];
+                    check = [
+                      {
+                        type = "tcp";
+                        port = "http";
+                        interval = "10s";
+                        timeout = "2s";
+                      }
+                    ];
+                  }
+                ];*/
+
+                task = rec {
+                  # ----------
+                  # Task: Marlowe-Runtime
+                  # ----------
+                  chainseekd = {
+                    env = {
+                      HOST = "127.0.0.1";
+                      PORT = "8090";
+                      QUERY_PORT = "8091";
+                      JOB_PORT = "8092";
+                      CARDANO_NODE_SOCKET_PATH = "/alloc/tmp/node.socket"; # figure out how to pass this from the cardano group
+                      WORKLOAD_CACERT = "/secrets/tls/ca.pem";
+                      WORKLOAD_CLIENT_KEY = "/secrets/tls/key.pem";
+                      WORKLOAD_CLIENT_CERT = "/secrets/tls/cert.pem";
+                    };
+
+                    template =
+                       _utils.nomadFragments.workload-identity-vault {inherit vaultPkiPath;}
+                      ++ _utils.nomadFragments.workload-identity-vault-consul {inherit consulRolePath;};
+
+                    config.image = ociNamer oci-images.chainseekd;
+                    config.ports = ["http"];
+                    user = "0:0";
+                    driver = "docker";
+                    kill_signal = "SIGINT";
+                    kill_timeout = "30s";
+                    resources.cpu = 2000;
+                    resources.memory = 4096;
+                    volume_mount = {
+                      destination = "/persist";
+                      propagation_mode = "private";
+                      volume = "persist-cardano-node-local";
+                    };
+                    vault = {
+                      change_mode = "noop";
+                      env = true;
+                      policies = ["marlowe-runtime"];
+                    };
+                  };
+                  marlowe-history = {
+                    env = {
+                      HOST = "127.0.0.1";
+                      PORT = 9080;
+                      QUERY_PORT = 9081;
+                      SYNC_PORT = 9082;
+                      CHAINSEEKD_HOST = "127.0.0.1";
+                      CHAINSEEKD_PORT = chainseekd.env.PORT;
+                      CHAINSEEKD_QUERY_PORT = chainseekd.env.QUERY_PORT;
+                    };
+
+                    config.image = ociNamer oci-images.marlowe-history;
+                    config.ports = ["http"];
```

Comment:
Does this task need to expose HTTP? What about the ports that the service listens on? How will we connect to those?